### ISSUE TITLE
Docs: in /create-medusa-app - Supabase IPv4 connection deprecated

### ISSUE THREAD

**Issue by LucasMatuszewski** [medusajs/medusa#6535]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

It looks like a connection to Supabase with direct DB access via IPv4 and pgBouncer doesn't work anymore (since January 26th 2024):
https://github.com/orgs/supabase/discussions/17817

I tried to install Medusa following this guide:
https://docs.medusajs.com/create-medusa-app#step-1-run-create-medusa-app

But the command from the Medusa example doesn't work anymore:
`npx create-medusa-app@latest --db-url postgresql://postgres:<password>@<host>.supabase.co:5432/postgres`

The new URI format is:
`postgres://postgres.<host>:<password>@aws-0-eu-central-1.pooler.supabase.com:5432/postgres`

But it didn't work when I tried it like this:
`npx create-medusa-app@latest --db-url postgres://postgres.<host>:<password>@aws-0-eu-central-1.pooler.supabase.com:5432/postgres`

I tried some other formats but neither works. I've tested the connection with other DB tools and all works.

Any ideas?

### How can this issue be resolved?

1. Probably this command should be supported: `npx create-medusa-app@latest --db-url postgres://postgres.<host>:<password>@aws-0-eu-central-1.pooler.supabase.com:5432/postgres`
...

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

**Comment by kailashv7** [comment]
I would like to work on this issue.

**Comment by salaxieb** [comment]
> I would like to work on this issue.

Any updates? it's very anticipated fix for me personally

**Comment by shahednasser** [comment]
Hey guys, will take some time to look at this maybe early next week. Thank you for your patience!

**Comment by KailashVenthan** [comment]
Hey  guys @salaxieb @shahednasser  
It works if .env is configured with connection string URI  found in supabase dashboard as follows

DATABASE_URL="postgres://[USERNAME]:[YOUR-PASSWORD]@aws-0-ap-south-1.pooler.supabase.com:5432/postgres

**Comment by shahednasser** [comment]
Hey guys, just an update: while investigating this it seems like it's an inconsistent problem, maybe due to connection errors or something else. On one run I faced errors, but all other runs I was able to finish it successfully.

As the `create-medusa-app` command isn't focused on supabase, we can't go too deep on looking for a consistent fix. So, for now I'll update the docs with the new URL format, but no further actions will be taken on the command itself.

For those facing errors, they can run the setup with the `--skip-db` option, then change the DB URL after the setup and re-run migrations.

EDIT: Also just wanna point out that the database URL must be wrapped with double quotes. Will include this in the docs as well
